react-frontend/src/components/DigitalFramework/digitalheader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../../css/pageheader.css';
+function DigitalHeader() {
+  return (
+    <div className="pageHeader">
```

Comment:
Semantically this should be a `<header>` ta

Reply:
Maybe it's just my naming, but I used header as its the first div on the page, not really a header persay, but I can change the name to prevent confusion